### PR TITLE
Vasua logging location fix

### PR DESCRIPTION
**Pull request by VivaanBahl**
fixed the logging to go to Alice/logs
We are making the decision right now to NOT check these in

### CODE REVIEW

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java [hunk 1]
```diff
@@ -28,12 +28,14 @@
...
-						+ logDir.getAbsolutePath());
-			}
+			
+			logDir.mkdirs();
+			
+//			if (!logDir.exists()) {
+//				logDir.mkdirs();
+//				System.out.println("Created directory: "
+//						+ logDir.getAbsolutePath());
+//			}
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java]: I think we should still have the create a dir, even if everyones computer should already have the dir.   Also we should not have dead code

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/mainFile.java [hunk 2]
```diff
@@ -114,7 +117,7 @@ public static void bringup_sim() throws Exception {
 		
 		// Set up the IMU
 		SerialPort sp = null;
-		String com = "COM4";//"COM18";
+		String com = "COM6";//"COM18";
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/mainFile.java]: Set this up as a static final

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/mainFile.java [hunk 3]
```diff
@@ -163,17 +166,17 @@ public void actionPerformed(String topicName, Message m) {
...
+//			sp = connect(com);
+//			System.out.println("DRIVE CONTROL connected to " + com);
+//		} catch (Exception e) {
+//			System.out.println("Unable to connect to necessary device on " + com);
+//			e.printStackTrace();
+//			throw new Exception("Device not found error");
+//		}
+//		drive_ctrl.setSerialPort(sp);
+//		sensorList.add(drive_ctrl);
 	}
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/mainFile.java]: No Dead code

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java (hunk 1 above)
Comment (msebek) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/RobotLogger.java]: Note that everyone's computer will only have the dir if there is a file placed in this directory. Git has no concept of an empty directory.

**Comment by VivaanBahl** [comment]
now #208

**Pull request closed without merging**